src/events/module_utils.js

```diff
@@ -53,15 +55,20 @@ module.exports.getModules = function getModules(id) {
         if (!autoNodePaths) {
             autoNodePaths = getAllNodeModulesPaths(rootFolder);
         }
+        utils.debugLog('Found the following paths', autoNodePaths);
         autoNodePaths.forEach((nodePath) => {
             if (!searchPaths.includes(nodePath)) {
                 searchPaths.push(nodePath);
             }
         });
     }
+    if (utils.isLambdaEnv && !searchPaths.includes(LAMBDA_DEFAULT_NODE_MODULES_PATH)) {
```

Comment:
this is the case if the user provides an external path to node_modules

Reply:
This is for all cases, where we are running inside Lambda - to make sure we're looking at /var/task/node-modules